kagami/database/models/worker.py

```diff
@@ -11,10 +11,18 @@ class WorkerRegStatus(enum.Enum):
     REGISTERED = "registered"
 
 
+class WorkerVerifyStatus(enum.Enum):
```

Comment:
I'm 100% sure I've seen exactly the same thing in other files.

Reply:
This one is not needed. Because all the worker stored in database is verified.